test.js

```diff
@@ -440,6 +440,14 @@ describe('cli', function () {
       commit('feat: first commit')
       execCli('--prerelease alpha').stdout.should.include('--tag alpha')
     })
+
+    it('does not advises use of --tag prerelease for private modules', function () {
```

Comment:
Still need to fix this: "does not advise"

Reply:
I had updated this but forgot to add it in my amended commit. (strike 2) 😬 Just forced pushed.